kong/core/cluster.lua

```diff
@@ -82,7 +82,7 @@ local function send_keepalive(premature)
       ngx_log(ngx.ERR, tostring(err))
     elseif #nodes == 1 then
       local node = nodes[1]
-      local _, err = singletons.dao.nodes:update(node, node, {ttl=singletons.configuration.cluster_ttl_on_failure})
+      local _, err = singletons.dao.nodes:update(node, node, {ttl=singletons.configuration.cluster_ttl_on_failure, quiet = true})
```

Comment:
Could we avoid long lines? As per our codestyle, not exceeding 80 chars width is ideal:
```lua
local _, err = singletons.dao.nodes:update(node, node, {
  ttl = singletons.configuration.cluster_ttl_on_failure, 
  quiet = true
})
```